gating/pre_merge_test/run_deploy.sh

```diff
@@ -47,8 +41,17 @@ elif [[ "${RE_JOB_SCENARIO}" == "ironic" ]]; then
   export DEPLOY_IRONIC="yes"
 fi
 
-if [[ ${RE_JOB_IMAGE} =~ loose$ ]]; then
+if [[ ${RE_JOB_IMAGE} =~ no_artifacts$ ]]; then
+  # Set the env var to disable artifact usage
+  export DEPLOY_ARTIFACTING="no"
+
+  # Upgrade to the absolute latest
+  # available packages.
+  apt-get update
+  DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade
```

Comment:
Line 61 uses DEBIAN_FRONTEND=noninteractive apt-get -y upgrade instead, I assume they should be the same, so can we align them?

Reply:
Using dist-upgrade requires a reboot to make everything active, so I don't really understand the point... but in this case I just moved the setting as it was.

Which would you like it to be to make it consistent?

Reply:
Probably upgrade, sounds like dist-upgrade can remove packages and may resolve dependencies which I’m not sure we need here.

Reply:
Done.